library/core/src/main/java/com/google/android/exoplayer2/scheduler/PlatformScheduler.java

```diff
@@ -86,6 +86,21 @@ public boolean cancel() {
     return true;
   }
 
+  @Override
+  public Requirements getSupportedRequirements(Requirements requirements) {
+    Requirements supportedRequirements = requirements;
+    if (Util.SDK_INT < 26) {
```

Comment:
What's this SDK_INT check for? I don't think it's supported on any API level?

Reply:
No need to fix this. I can do it when I merge. Just confirming that the comment is correct is fine :).

Reply:
Good catch, that is correct.